civihr_default_theme/scss/overrides/_datepicker-civicrm.scss

```diff
@@ -7,16 +7,11 @@
 // Override styles for civicrm datepicker specific to SSP
 .ui-datepicker {
   .ui-datepicker-calendar thead tr th {
-    font-size:13px !important;
+    font-size: 11.9px !important; //Calendar heading font size as in SSP
   }
-
-  tbody {
-    border-top: 0;
-  }
-
   .ui-datepicker-month,
   .ui-datepicker-year {
-    font-size: 15px !important;
+    font-size: 14px !important; // Select option font size as in SSP
```

Comment:
Why these are changed, because these styles are for civicrm right?(that is what I can understand from the partial name) then how are they related to onboarding styles? 
Could you post a screenshot showing the difference without these changes?

Reply:
Without these overrides, the date-picker looks like this:
<img width="332" alt="screen shot 2017-12-26 at 12 35 02 pm" src="https://user-images.githubusercontent.com/6307362/34349467-5a4bba38-ea39-11e7-9020-f71990d8d3a3.png">

See the font size are small. The styles are form shoreditch but does not match with that of SSP.